rmgpy/data/kinetics/family.py

```diff
@@ -2170,8 +2166,9 @@ def _generate_reactions(self, reactants, products=None, forward=True, prod_reson
         elif len(reactants) == 3 and len(template_reactants) == 3:
             """
             This could be a surface reaction
-                A + X + X <=> BX + CX  (dissociative adsorption)
-                A + X + X <=> AXX      (bidentate adsorption)
+                A + X + X <=> BX + CX    (dissociative adsorption)
+                A + X + X <=> AXX        (bidentate adsorption)
+                ABX + X + X <=> AXX + BX (bidentate dissociation)
```

Comment:
Since there is another family called bidentate dissociation, I think it would be better to have "dissociation to bidentate" in the parentheses here for consistency with the name in the reaction family database.